upgrade/upgrade-2.1.0.php

```diff
@@ -27,7 +26,7 @@
  *
  * @return bool
  */
-function upgrade_module_2_0_1($module)
+function upgrade_module_2_1_0($module)
```

Comment:
Are you sure?

Reply:
Yes, this upgrade file has been added yesterday and version 2.0.1 never released

Reply:
ohh I understand :)